.travis.yml

```diff
@@ -1,5 +1,13 @@
 sudo: false
```

Comment:
This must be `required` in order for trusty to work properly (according to travis docs).